internal/storage/bucket_handle_test.go

```diff
@@ -256,3 +257,94 @@ func (t *BucketHandleTest) TestCreateObjectMethodWhenGivenGenerationObjectNotExi
 	AssertEq(nil, obj)
 	AssertTrue(strings.Contains(err.Error(), "Error 412: Precondition failed"))
 }
+
+func (t *BucketHandleTest) TestGetProjectValueWhenGcloudProjectionIsNoAcl() {
+	proj := getProjectionValue(gcs.Projection(storage.ProjectionNoACL))
```

Comment:
I didnt follow. what are you trying to do here. you need to  pass gcs.Projection type to getProjectionValue method. 

Why are you creating a gcs.Projection using storage.Projection?

Reply:
fixed